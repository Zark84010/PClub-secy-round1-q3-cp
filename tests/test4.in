100 42 50
14616 2496 6768 16182 2200 3528 10920 3456 2214 5402 1656 4080 5146 3024 4896 180 2714 9520 10716 1032 4440 880 6460 4950 2272 756 14904 8436 8946 15908 312 4760 444 2820 1624 4800 7636 2040 4550 4488 1584 12166 15130 416 11000 4026 10626 1040 4698 7600 13632 5530 272 588 3348 6042 324 7308 11880 1800 2460 1152 14112 12464 560 7752 10584 522 396 42 504 120 150 174 432 198 180 24 120 426 414 54 96 246 300 582 144 114 276 282 312 132 336 480 534 390 174 252 306 564 
c 65 15
f 15 54
f 57 96
c 66 60
f 37 45
f 52 52
c 8 105
f 22 77
f 96 100
c 94 57
f 29 68
f 12 30
c 5 12
f 64 77
f 39 61
c 19 27
f 89 98
f 18 51
c 44 93
f 84 90
f 100 100
c 91 78
f 40 45
f 87 98
c 65 114
f 8 41
f 5 97
c 29 60
f 44 75
f 69 73
c 11 18
f 2 15
f 31 59
c 37 15
f 27 29
f 66 99
c 59 15
f 38 53
f 25 81
c 100 60
f 51 69
f 72 83
c 31 42
f 95 99
f 64 92
c 97 120
f 74 93
f 69 91
c 67 15
f 41 51
