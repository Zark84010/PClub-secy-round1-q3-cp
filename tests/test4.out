82
42
82
226730
42
246
42
82
42
82
42
42
246
23124
42
42
42
42
42
246
42
42
42
42
42
42
42
246
246
42
42
42
42
